totalopenstation/formats/landxml.py

```diff
@@ -362,4 +368,189 @@ def to_string(self):
         self.root.set("date", time.strftime("%Y-%m-%d"))
         self.root.set("time", time.strftime("%H:%M:%S"))
         pretty_xml = _indent(self.root)
-        return xml.tostring(pretty_xml)
+        return xml.tostring(pretty_xml).decode()
+
+
+class FormatParser(Parser):
+    """
+    A FormatParser for LandXML data format.
+
+    As the model data is in LandXML format, only Survey tags is kept.
+
+    It doesn't inherit from the base Parser class because the internal
+    procedure is quite different, but it implements the same API so it
+    can work nicely with other parts of the library.
+    """
+
+    def __init__(self, data):
+        data = re.sub('(\\n|\\t)', '', data)
+        self.line = xml.fromstring(data) 
+
+    def _points(self):
+        '''Compute raw data to get points coordinates.
+
+        This parser is based on the information in :ref:`if_leica_gsi`
```

Comment:
the reference to leica_gsi must be updated

Reply:
This method is no more used.